game/scripts/npc/abilities/brewmaster_primal_split.txt

```diff
@@ -38,7 +38,7 @@
       "01"
       {
         "var_type"                                        "FIELD_FLOAT"
-        "duration"                                        "16 18 20 20 20"
+        "duration"                                        "16 18 20 22 24"
```

Comment:
Leave this at 20 seconds at OAA levels. This was intentially done by somebody because with octarine max level at level 50 you have 100% uptime on Primal Split. Making Bremaster unkillable with proper micro.